packages/autorest.typescript/src/restLevelClient/transforms/transformParameterTypes.ts

```diff
@@ -304,4 +304,4 @@ function getAllOperations(model: CodeModel): Operation[] {
   }
 
   return operations;
-}
+}
```

Comment:
I didn't support header with array case in swagger side, because I didn't find a valid case anywhere and when I try to build one the autorest would complain `Unknown primitive schema array`.